extensions/redis-client/runtime/pom.xml

```
@@ -149,5 +149,12 @@
                 <redis.base.image>redis:6</redis.base.image>
             </properties>
         </profile>
+
+        <profile>
+            <id>redis-7.0</id>
+            <properties>
+                <redis.base.image>redis/redis-stack:7.0.6-RC9</redis.base.image>
```

```suggestion
                <redis.base.image>redis/redis-stack:7.2.0-v0</redis.base.image>
```

We can use `redis/redis-stack:7.2.0-v0` here?